server/services/prismic-parsers.js

```diff
@@ -399,17 +399,19 @@ export function asText(maybeContent: any) {
 }
 
 export function asHtml(maybeContent: any) {
-  // Prismic can send us empty html elements which can lead to unwanted UI in templates.
-  // Check that `asText` wouldn't return an empty string.
-  const isEmpty = !maybeContent || asText(maybeContent).trim() === '';
-
-  return isEmpty ? null : RichText.asHtml(maybeContent, linkResolver).trim();
+  return isMissingOrEmpty(maybeContent) ? null : RichText.asHtml(maybeContent, linkResolver).trim();
```

Comment:
Could we use this?
https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/services/prismic/parsers.js#L57

Either here, or at the implementation?

Reply:
We spoke. Added `parseRichText` to the common parsers.